src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -111,9 +94,9 @@ public void GetItem(int size)
                 using (iteration.StartMeasurement())
                     for (int i = 0; i <= 10000; i++)
                     {
-                        retrieved = dict["key1"]; retrieved = dict["key2"]; retrieved = dict["key3"];
-                        retrieved = dict["key4"]; retrieved = dict["key5"]; retrieved = dict["key6"];
-                        retrieved = dict["key7"]; retrieved = dict["key8"]; retrieved = dict["key9"];
+                        retrieved = dict[1]; retrieved = dict[2]; retrieved = dict[3];
```

Comment:
It is probably a good idea.  When there are collisions insertion and access is slower as we have to walk the buckets array.